lib/x11utils.pm

```diff
@@ -408,6 +409,18 @@ sub turn_off_gnome_screensaver_for_gdm {
     script_run 'sudo -u gdm dbus-launch gsettings set org.gnome.desktop.session idle-delay 0';
 }
 
+=head2 turn_off_gnome_screensaver_for_running_gdm
+
+turn_off_gnome_screensaver_for_running_gdm()
+
+Disable screensaver in gnome for running gdm. The function should be run under root. To be called
+from a command prompt, for example an xterm window.
+
+=cut
+sub turn_off_gnome_screensaver_for_running_gdm {
```

Comment:
Why not put logic into one function turn_off_gnome_screensaver_for_gdm?

Reply:
This is different functions, the new one is for running gdm, this suggestion comes from desktop team.